### ISSUE TITLE
How to print sub-items of a key?

### ISSUE THREAD

**Issue by BlueFalconHD** [trentm/json#152]
I have a file, assets.json:
```
{
 "window":{
         "bottom":"_"
 }
}
```

I tried to cat the file while piping it with JSON:
`cat assets.json | json window bottom`
which outputs:
```
{
  "bottom": "_"
}
```
I want it to just output the value of the  key `bottom`.

Like:
`_`

**Comment by rhettjay** [comment]
`cat assets.json | json window.bottom`

**Comment by BlueFalconHD** [comment]
Really helpful! (Thanks for responding 2 years after my issue!!!)

**Comment by rhettjay** [comment]
Of course! I was looking through the issues searching for an answer to an issue I'm coming across with arrays. Figured might as well help anyone who runs into the issue.
